packages/canary-react/src/stories/ic-table-of-contents.stories.js

```diff
@@ -0,0 +1,263 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React, { useState } from "react";
+import { IcTopNavigation, IcNavigationItem, IcPageHeader, IcTypography, IcButton } from "@ukic/react";
+import { IcTableOfContents } from '../components';
+import readme from "../../../canary-web-components/src/components/ic-table-of-contents/readme.md";
+
+export default {
+  title: "React Components/Table of Contents",
+  component: IcTableOfContents,
+  parameters: {
+    componentAPI: {
+      data: readme,
+    },
+  },
+};
+
+/**
+ * The table of contents component is used to create a navigable list of sections within a page or document.
+ * 
+ * It can be positioned on the left, right or on top of the child content.
```

Comment:
I think we're just putting all canary guidance onto the website now, rather than in Storybook, following [#1557](https://github.com/mi6/ic-design-system/issues/1557). So I guess this comment can be removed?

Also the comment in the web components Storybook. Although happy for the comments to be left  if you think they're still useful - but just wanted to point out that it will probably just be our team who will see them :)